pennylane/grouping/graph_colouring.py

```diff
@@ -36,7 +36,17 @@ def largest_first(binary_observables, adj):
     """Performs graph colouring using the Largest Degree First heuristic. Runtime is quadratic in
     number of vertices.
 
-    **Usage example:**
+    Args:
```

Comment:
In such cases, the `Example` section was placed at the end of the docstring (GitHub diffs it odd)